test/e2e/specs/editor/various/navigable-toolbar.spec.js

```diff
@@ -92,13 +99,25 @@ describe( 'Block Toolbar', () => {
 			await page.keyboard.type( 'Example' );
 
 			// Upward.
-			await pressKeyWithModifier( 'alt', 'F10' );
+			await pageUtils.pressKeys( 'alt+F10' );
 
-			expect(
-				await page.evaluate( () => {
-					return document.activeElement.getAttribute( 'aria-label' );
-				} )
-			).toBe( 'Show document tools' );
+			await expect(
+				page.locator( 'role=toolbar[name="Document tools"i]' )
+			).toBeVisible();
 		} );
 	} );
 } );
+
+class navigableToolbar {
```

Comment:
Classes in the other e2e tests capitalize the class names, so that's best to use that format here too for consistency: `NavigableToolbar`.

Reply:
I think we can drop the whole utility class here and replace `isInBlockToolbar` with this assertion, as I think it's more descriptive.

```js
await expect(
  page
	  .getByRole( 'toolbar', { name: 'Block Tools' } )
	  .getByRole( 'button', { name: 'Paragraph' } )
).toBeFocused();
```